.github/workflows/ci.yml

```
@@ -46,12 +46,16 @@ jobs:
       run: |
         poetry env use ${{ matrix.python-version }}
         poetry install --with dev
+        echo "JANUS_PATH=$(poetry run which janus)" >> $GITHUB_ENV
 
     - name: Run test suite
+
       env:
-        # show timings of tests
+        # show timings of test
         PYTEST_ADDOPTS: "--durations=0"
-      run: poetry run pytest --cov aiida_mlip --cov-append .
+        JANUS_PATH: ${{ env.JANUS_PATH }}
+      run: |
+        poetry run pytest --cov aiida_mlip --cov-append .
```

```suggestion
      run: poetry run pytest --cov aiida_mlip --cov-append .
```
Very minor (and I think introduced by me) but looks a bit messy